library/core/src/main/java/com/google/android/exoplayer2/audio/Ac3Util.java

```diff
@@ -302,6 +313,19 @@ private static int getAc3SyncframeSize(int fscod, int frmsizecod) {
     }
   }
 
+  public static int parseTrueHDSyncframeAudioSampleCount(ByteBuffer buffer) {
+
+    if (buffer.limit() < 12)
+      return 0;
```

Comment:
Use C.INDEX_UNSET instead of 0?

Reply:
Sure, but I think I have to add a check for framesPerEncodedSample == 0 || framesPerEncodedSample == C.INDEX_UNSET where we call getFramesPerEncodedSample around Line 747.  Or we can do a framesPerEncodedSample <= 0 which seems bad.